client/state/data-layer/wpcom/comments/index.js

```diff
@@ -20,6 +21,16 @@ import { errorNotice, successNotice } from 'state/notices/actions';
 import { getSitePost } from 'state/posts/selectors';
 import { getPostOldestCommentDate, getPostNewestCommentDate } from 'state/comments/selectors';
 import getSiteComment from 'state/selectors/get-site-comment';
+import { decodeEntities } from 'lib/formatting';
+
+const commentsFromApi = comments =>
+	comments.map( comment => ( {
```

Comment:
probably should validate that `comments` is truthy (and maybe has `map`?) before calling `map` since this just coming off the wire.

Reply:
switching to _